diff --git a/SDL2-2.26.3/include/SDL_cpuinfo.h b/SDL2-2.26.3/include/SDL_cpuinfo.h
index ed5e979..4e4078a 100644
--- a/SDL2-2.26.3/include/SDL_cpuinfo.h
+++ b/SDL2-2.26.3/include/SDL_cpuinfo.h
@@ -33,7 +33,7 @@
 /* Need to do this here because intrin.h has C++ code in it */
 /* Visual Studio 2005 has a bug where intrin.h conflicts with winnt.h */
 #if defined(_MSC_VER) && (_MSC_VER >= 1500) && (defined(_M_IX86) || defined(_M_X64))
-#ifdef __clang__
+#if defined(__clang__) && !_SDL_HAS_BUILTIN(_m_prefetch)
 /* As of Clang 11, '_m_prefetchw' is conflicting with the winnt.h's version,
    so we define the needed '_m_prefetch' here as a pseudo-header, until the issue is fixed. */
 
diff --git a/SDL2-2.26.3/include/SDL_endian.h b/SDL2-2.26.3/include/SDL_endian.h
index 582c3a8..afbd9fe 100644
--- a/SDL2-2.26.3/include/SDL_endian.h
+++ b/SDL2-2.26.3/include/SDL_endian.h
@@ -33,7 +33,7 @@
 #if defined(_MSC_VER) && (_MSC_VER >= 1400)
 /* As of Clang 11, '_m_prefetchw' is conflicting with the winnt.h's version,
    so we define the needed '_m_prefetch' here as a pseudo-header, until the issue is fixed. */
-#ifdef __clang__
+#if defined(__clang__) && !_SDL_HAS_BUILTIN(_m_prefetch)
 #ifndef __PRFCHWINTRIN_H
 #define __PRFCHWINTRIN_H
 static __inline__ void __attribute__((__always_inline__, __nodebug__))
